fedora_messaging/tests/unit/test_session.py

```diff
@@ -292,6 +293,30 @@ def test_external_auth(self):
             consumer._parameters.credentials, credentials.ExternalCredentials
         )
 
+    def test_url_with_client_properties(self):
+        """Assert when URL contains client props, values are not taken from config."""
+        test_url = "amqps://username:password@rabbit.example.com/vhost?client_properties={'k':'v'}"
+        tls_conf = {
```

Comment:
Since this test is all about the client properties in the URL, I think you can drop the tls config and use an "amqp://..." URL, and drop the assertions about the host, port, and SSL options. Having a test for each of those things is nice because when it fails it's very obvious what part of the URL parsing changed or is broken. The fewer assertions in a test case, the better (generally speaking).